### ISSUE TITLE
"Latest" release is from 2018

### ISSUE THREAD

**Issue by TheEvilSkeleton** [Eloston/ungoogled-chromium#1414]
Can you fix this?

![image](https://user-images.githubusercontent.com/50847364/109447003-bf26f980-7a10-11eb-8ace-662552b29bdb.png)

**Comment by jacob755** [comment]
click on +118 releases see is updated
to download binaries
https://ungoogled-software.github.io/ungoogled-chromium-binaries/

**Comment by TheEvilSkeleton** [comment]
@jacob755 I'm aware of that, but it might give the wrong idea to people that are just taking a look at the repo.

**Comment by hefty-fading-culprit-epiphany-drew** [comment]
> @jacob755 I'm aware of that, but it might give the wrong idea to people that are just taking a look at the repo.


Yes! When I first saw this GitHub repo and found the lastest release to be 1 yo I thought the developement was halted

**Comment by Eloston** [comment]
I tried looking a bit into this, there are a lot of bloated node.js-based solutions. Ideally I'd need something that:

1. Is very barebones, just creates empty releases whenever a tag is pushed. No extra features
2. Does not involve a complex dependency system, e.g. npm
3. Can be self-hosted, i.e. something we can fork into the ungoogled-software organization and maintain ourselves to guarantee safety and security.

...but none of the existing solutions I found fit all three criterion. I suppose we might have to create our own.
